source/navitiacommon/navitiacommon/default_values.py

```diff
@@ -116,6 +116,22 @@
 # Number of greenlets simultaneously used for Real-Time proxies calls
 realtime_pool_size = 3
 
+# Minimum number of different suggested journeys
+min_nb_journeys = 0
+
+# Minimum number of calls to kraken
+min_journeys_calls = 1
+
+# Maximum number of successive physical modes for an itinerary
+max_successive_physical_mode = 100
```

Comment:
100?

Reply:
I am looking for a default value infinity. Do you have any suggestion?